charts/linkerd2/README.md

```diff
@@ -141,6 +140,7 @@ Kubernetes: `>=1.13.0-0`
 | global.clusterNetworks | string | `"10.0.0.0/8,100.64.0.0/10,172.16.0.0/12,192.168.0.0/16"` | The cluster networks for which service discovery is performed. This should include the pod network but need not include the node network. By default, all private networks are specified so that resolution works in typical Kubernetes environments. |
 | global.cniEnabled | bool | `false` | enabling this omits the NET_ADMIN capability in the PSP and the proxy-init container when injecting the proxy; requires the linkerd-cni plugin to already be installed |
 | global.controlPlaneTracing | bool | `false` | control plane trace configuration |
+| global.controlPlaneTracingNamespace | string | `"linkerd-jaeger"` |  |
```

Comment:
I know it's pretty clear what this is, but add a description here to stay consistent with everything else?